src/Twig/SourceCodeExtension.php

```diff
@@ -28,11 +30,22 @@
  */
 final class SourceCodeExtension extends AbstractExtension
 {
+    private string|FileLinkFormatter|array|false $fileLinkFormat;
```

Comment:
this type looks weird to me given that the constructor enforces FileLinkFormatter

Reply:
Little leftover, fixed, thank you.